apps/testing/src/utils/paramsBuilder.ts

```diff
@@ -28,7 +28,11 @@ export const useConfigParams = (initParams: InitialParams): ParamsAsProps => {
     accessToken: searchParams.get('accessToken') || initParams.accessToken,
     allowProfileEdit: parseValue(searchParams.get('enableProfileEdit')) ?? true,
     isMultipleFilesMessageEnabled: parseValue(searchParams.get('enableMultipleFilesMessage')) ?? true,
-    uikitOptions: {},
+    uikitOptions: {
+      groupChannel: {
+        enableMarkdownForUserMessage: true,
+      }
+    },
```

Comment:
Could you revert this?

And please add `groupChannel_ enableMarkdownForUserMessage` to `const paramKeys = [...]` below.

Reply:
done!